tests/lightning_qubit/test_adjoint_jacobian_class.py

```diff
@@ -138,6 +156,35 @@ def test_empty_measurements(self, lightning_sv):
         jac = self.calculate_jacobian(lightning_sv(num_wires=3), tape)
         assert len(jac) == 0
 
+    def test_empty_trainable_params(self, lightning_sv):
```

Comment:
These tests were added to increase the coverage of the code

Reply:
Great job!